sktime/forecasting/tests/test_all_forecasters.py

```diff
@@ -61,13 +61,24 @@
 def test_get_fitted_params(Forecaster):
     """Test get_fitted_params."""
     f = _construct_instance(Forecaster)
-    f.fit(y_train, fh=FH0)
-    try:
-        params = f.get_fitted_params()
-        assert isinstance(params, dict)
+    if f.get_tag("scitype:y") in ["univariate", "both"]:
```

Comment:
One option to remove some of the code duplication may be to write something like the following: 

```python
n_columns = _get_n_columns(f.get_tag("y:scitype"))

for n_columns in n_columns_list:
    y_train = _make_series(n_columns=n_columns)
    f.fit(y_train, fh=FH0)
    try:
        params = f.get_fitted_params()
        assert isinstance(params, dict)

    except NotImplementedError:
        pass


def _get_n_columns(tag):
    if tag == "univariate":
        n_columns_list = [1]
    elif tag == "multivariate":
        n_columns_list = [2]
    # else
    elif tag == "both":
        n_columns_list = [1, 2]
    return n_columns_list
```